Tests fail locally due to path differences

~~~~{.bash}
======================================================================
FAIL: test_loggedexception_location (test.exceptions.ExceptionsTest)
Test inclusion of location information in log message for LoggedException.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/ageorges/work/hpc/repos/vsc-base/test/exceptions.py", line 161, in test_loggedexception_location
    self.assertTrue(log_re.match(logtxt), "%s matches %s" % (log_re.pattern, logtxt))
AssertionError: ^setup :: BOOM \(at vsc/install/testing.py:[0-9]+ in assertErrorRegex\)$ matches setup :: BOOM (at vsc/lib/python2.7/site-packages/vsc_install-0.9.10-py2.7.egg/vsc/install/testing.py:121 in assertErrorRegex)


----------------------------------------------------------------------
~~~~

--- a/test/exceptions.py
+++ b/test/exceptions.py
@@ -156,7 +156,7 @@ class ExceptionsTest(TestCase):
         self.assertErrorRegex(LoggedException, 'BOOM', raise_testexception, 'BOOM')
         logToFile(tmplog, enable=False)
 
-        log_re = re.compile(r"^%s :: BOOM \(at vsc/install/testing.py:[0-9]+ in assertErrorRegex\)$" % rootlogname)
+        log_re = re.compile(r"^%s :: BOOM \(at (?:.*?/)?vsc/install/testing.py:[0-9]+ in assertErrorRegex\)$" % rootlogname)
         logtxt = open(tmplog, 'r').read()
         self.assertTrue(log_re.match(logtxt), "%s matches %s" % (log_re.pattern, logtxt))
 
@@ -170,7 +170,7 @@ class ExceptionsTest(TestCase):
         self.assertErrorRegex(LoggedException, 'BOOM', raise_testexception, 'BOOM')
         logToFile(tmplog, enable=False)
 
-        log_re = re.compile(r"^%s :: BOOM \(at .*/vsc/install/testing.py:[0-9]+ in assertErrorRegex\)$" % rootlogname)
+        log_re = re.compile(r"^%s :: BOOM \(at (?:.*?/)?vsc/install/testing.py:[0-9]+ in assertErrorRegex\)$" % rootlogname)
         logtxt = open(tmplog, 'r').read()
         self.assertTrue(log_re.match(logtxt), "%s matches %s" % (log_re.pattern, logtxt))
 

